CHANGELOG.md

```diff
@@ -9,6 +9,10 @@
   - Made associated classes `ContributorStatistics` and 
     `ContributorWeekStatistics` immutable. Since these classes are only meant as
     return values, we're not treating this as a breaking change.
+- Added `Stream<CodeSearchResults> github.search.code(...)` search API
```

Comment:
Since there are *new* changes, this should be 4.1, right?

Reply:
Yeah, I suppose it should be 4.1. I'll bump the version in this PR. 
Whoever merges are you ok using the github squash or rebase and merge functionality or would you rather I squash and forcepush?
![image](https://user-images.githubusercontent.com/6053699/50946617-7b9b5b00-1457-11e9-98d4-f50d0d8d55cf.png)